<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QMainWindow" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>366</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>778</width>
    <height>366</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>778</width>
    <height>366</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SRSystem</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="spechText">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>0</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="mngSpeechBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>381</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>Manage speech</string>
    </property>
    <widget class="QPushButton" name="toogleSRBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Listening</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainBtn">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Train</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="powerBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>361</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="sampleGLayout"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>361</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="debugGLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionNewDatabase"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import...</string>
   </property>
  </action>
  <action name="actionNew_database">
   <property name="text">
    <string>New database</string>
   </property>
  </action>
  <action name="actionImport_2">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNewDatabase">
   <property name="text">
    <string>Manage database</string>
   </property>
  </action>
  <action name="actionImportDatabase">
   <property name="text">
    <string>Import database</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
